renderer/components/Wallet/WalletButtons.js

```diff
@@ -1,23 +1,32 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Box } from 'rebass/styled-components'
-import { FormattedMessage } from 'react-intl'
+import { FormattedMessage, injectIntl } from 'react-intl'
 import { Button } from 'components/UI'
 import messages from './messages'
+import { intlShape } from '@zap/i18n'
 
-const WalletButtons = ({ openModal }) => (
+const WalletButtons = ({ intl, openModal, totalBalance }) => (
   <Box as="section">
     <Button mr={2} onClick={() => openModal('PAY_FORM')} width={145}>
       <FormattedMessage {...messages.pay} />
     </Button>
-    <Button onClick={() => openModal('REQUEST_FORM')} width={145}>
+    <Button
+      className="hint--bottom-left"
+      data-hint={totalBalance ? null : intl.formatMessage({ ...messages.request_disabled_memo })}
+      disabled={!totalBalance}
```

Comment:
You are looking for `isDisabled` prop here i believe

Reply:
Done.